pkg/statistics/handle/syncload/stats_syncload.go

```diff
@@ -105,7 +105,11 @@ func (s *statsSyncLoad) SendLoadRequests(sc *stmtctx.StatementContext, neededHis
 		localItem := item
 		resultCh := globalStatsSyncLoadSingleFlight.DoChan(localItem.Key(), func() (any, error) {
 			timer := time.NewTimer(timeout)
-			defer timer.Stop()
```

Comment:
Better to use "time.After"?

Reply:
We can use it late. but this PR need to cherrypick into other version. It’s best to keep the changes minimal.